### PR TITLE
Use guzzle6

### PR DESCRIPTION
**Pull request by dawehner**
Once the gastonjs guzzle6 PR is merged we could use this one.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -51,5 +51,11 @@
     "branch-alias": {
       "dev-master": "0.4.x-dev"
     }
-  }
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/dawehner/gastonjs.git"
```
Comment (jibran) [inline composer.json]: I think this is unrelated change.

Comment (dawehner) [inline composer.json]: Well, we need to wait until the other PR, otherwise I could not test it locally to ensure it actually works

**Comment by stof** [comment]
See https://github.com/jcalderonzumba/MinkPhantomJSDriver/pull/46 for a better way to handle it. The driver does not use guzzle directly, and so does not need a requirement on it.

**Comment by dawehner** [comment]
Sure let's drop the dependency as it is

**Comment by jcalderonzumba** [comment]
Closing issue now.

**Comment by dawehner** [comment]
Cool, thank you
